pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -471,6 +471,9 @@ type QuotaSpec struct {
 	// MaxBytes represents the quota in bytes
 	MaxBytes *uint64 `json:"maxBytes,omitempty"`
 
+	// MaxBytes represents the quota in bytes as a string
+	MaxBytesHuman *string `json:"maxBytesHuman,omitempty"`
```

Comment:
How about if we name this maxSize, and then add a comment that `maxBytes` is no longer preferred? To keep backward compatibility we would still officially support `maxBytes`, but documentation would only mention the `maxSize`.
```suggestion
	MaxSize *string `json:"maxSize,omitempty"`
```

Reply:
change made